.ci/jenkins/Jenkinsfile.staging-build

```
@@ -795,6 +801,23 @@ pipeline {
                 }
             }
         }
+
+        stage('STAGING: Push kie-sandbox-helm-chart to quay.io') {
+            when {
+                expression { !params.DRY_RUN }
+            }
+            steps {
+                dir('kie-tools') {
+                    script {
+                        helmUtils.pushChartToRegistry(
+                            "${env.KIE_SANDBOX_HELM_CHART__registry}/${env.KIE_SANDBOX_HELM_CHART__account}",
+                            "packages/${env.KIE_SANDBOX_HELM_CHART__name}/dist/${env.KIE_SANDBOX_HELM_CHART__name}-${env.KIE_SANDBOX_HELM_CHART__tag}.tgz",
```

```suggestion
                            "packages/kie-sandbox-helm-chart/dist/${env.KIE_SANDBOX_HELM_CHART__name}-${env.KIE_SANDBOX_HELM_CHART__tag}.tgz",
```

The package name won't change based on the environment variable.